### ISSUE TITLE
[iPad/Mac]a11y_Datagrid> selectable> A11y> 1 violations & 1 incomplete is appearing

### ISSUE THREAD

**Issue by Uditsingh67** [cengage/react-magma#877]
**Describe the bug**
Datagrid>> "1 violations & 1 incomplete " is appearing for "selectable".

**To Reproduce**
**Steps to reproduce the behavior:**

1. Go to 'https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/datagrid--selectable
2. Click on 'Accessibility" add on
3. "1 violations & 1 incomplete " is appearing

Expected behavior
"0 violations & 0 incomplete" should appear for datagrid> selectable.

**QA note:** 
**Issue is also reproducible for components mention below :**
1. disabled-selectable-row
2. pagination-changed-defaults
3. controlled-pagination 
4. without-pagination
5. pagination-with-custom-component

<html>
<body>

OS | Version | Result
-- | -- | --
Win 10 (Chrome) | 102.0.5005.63 | (http://somup.com/c3jbiRu93S)
Win 10 (Firefox) | 101.0 | (http://somup.com/c3jbjnu90X)
Win 10 (Edge) | 103.0.1264.37 | (http://somup.com/c3jbjfu90F)
MAC (safari) | 15.3 | (http://somup.com/c3jbjcu901)
Ipad (safari) | v15.1 | (http://somup.com/c3jbiLu93C)

</body>
</html>

**Comment by Uditsingh67** [comment]
The issue is fixed :
**See Screen-cast :** 

- Mac : http://somup.com/c3jZiPUhDs
- iPad : http://somup.com/c3jZiRUhbn
Thanks!

**Comment by Uditsingh67** [comment]
Issue does not exist.
See Screen-cast : http://somup.com/c3jweVU2lN
